CHANGELOG.md

```diff
@@ -1,1200 +1,8 @@
 # Change Log
 
-## [0.1.20](https://github.com/kubeflow/pipelines/tree/0.1.20) (2019-05-14)
-[Full Changelog](https://github.com/kubeflow/pipelines/compare/0.1.19...0.1.20)
+## [0.1.13](https://github.com/jingzhang36/pipelines/tree/0.1.13) (2019-03-22)
```

Comment:
Weird, it is supposed to generate changelog for 0.1.21.

Reply:
Also the repo is wrong

Reply:
Might be because my local master branch is messed up :(